client/dashboard/task-list/tasks/payments/index.js

```diff
@@ -444,6 +468,26 @@ class Payments extends Component {
 				content: <Fragment />,
 				visible: ! showIndividualConfigs,
 			},
+			{
+				key: 'payfast',
+				label: __( 'Enable PayFast', 'woocommerce-admin' ),
+				description: __(
+					'Connect your store to your PayFast account',
+					'woocommerce-admin'
+				),
+				content: (
+					<PayFast
+						manualConfig={ manualConfig }
```

Comment:
Are we using any of these props in the `PayFast` component?

Reply:
No, overly eager copying ;-)